Package.swift

```diff
@@ -6,10 +6,20 @@ let package = Package(
 	products: [
 		.library(
 			name: "AssimpKit",
-			targets: ["AssimpKit"]
+			targets: ["AssimpKitSPM"]
```

Comment:
PP: more descriptive name: AssimpKitDynamic ?

PP: expose both the static and dynamic versions as different libraries